AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -10889,6 +10889,7 @@ stokke.com##.luana-module-cookie-policy
 bitburger.de,otto.de##.m-cookie-layer
 m.krombacher.de##.mainContent > .textSection + .formSection + .textSection
 mini.de##.md-cookie
+jovemnerd.com.br##div:matches-css(z-index: 999):remove()
```

Comment:
Too generic.

`jovemnerd.com.br##footer + div[class=""][style*="background-color"][style*="position: fixed"]`

Reply:
Also note `#$?#div:matches-css(z-index: 999) { remove: true; }` is our canonical syntax for the expression. However, avoid using `:matches-css()` or pseudo-class `remove` unless absolutely necessary.

Reply:
Done